typescript/sdk/src/core/CoreDeployer.hardhat-test.ts

```diff
@@ -93,7 +93,8 @@ describe('core', async () => {
       try {
         await deployer.deploy(coreConfig);
       } catch (e: any) {
-        expect(e.message).to.include('Timed out in 1ms');
```

Comment:
How about `it.skip` instead?

Reply:
the test is still useful for catching the error but for some reason the message is empty now